guard/tests/validate.rs

```diff
@@ -682,18 +688,19 @@ mod validate_tests {
         assert_eq!(StatusCode::INTERNAL_FAILURE, status_code);
     }
 
-    #[test]
-    fn test_junit_without_structured_flag() {
+    #[rstest::rstest]
+    #[case("junit")]
+    #[case("sarif")]
+    fn test_outputs_without_structured_flag(#[case] output: &str) {
```

Comment:
Minor: I think we can use a better name here, something like `test_structured_outputs_fail_without_structured_flag`, i know this is kinda long/verbose but i think it provides appropriate context.